--- setup.py	2025-10-09 17:33:28.000000000 -0600
+++ setup.py	2025-10-09 17:32:32.000000000 -0600
@@ -1,9 +1,5 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
 """SPORCO package configuration."""
 
-from __future__ import print_function
 from builtins import next
 from builtins import filter
 
@@ -21,10 +17,11 @@
 with open(os.path.join(name, '__init__.py')) as f:
     version = parse(next(filter(
         lambda line: line.startswith('__version__'),
-        f))).body[0].value.s
+        f))).body[0].value.value
 
 packages = ['sporco', 'sporco.prox', 'sporco.admm', 'sporco.pgm',
-            'sporco.dictlrn']
+            'sporco.dictlrn', 'sporco.cuda', 'sporco.data',
+            'sporco.cupy', 'sporco.cupy.pgm', 'sporco.cupy.admm', 'sporco.cupy.dictlrn']
 
 docdirbase = 'share/doc/%s-%s' % (name, version)
 data = [(os.path.join(docdirbase, 'examples/scripts'),
@@ -71,7 +68,7 @@
                         'Convolutional Sparse Coding', 'Optimization',
                         'ADMM', 'PGM'],
     platforms        = 'Any',
-    license          = 'BSD',
+    license          = 'BSD-3-Clause',
     url              = 'https://github.com/bwohlberg/sporco',
     author           = 'Brendt Wohlberg',
     author_email     = 'brendt@ieee.org',
@@ -79,8 +76,6 @@
     package_data     = {'sporco': ['data/*.png', 'data/*.jpg', 'data/*.npz']},
     data_files       = data,
     include_package_data = True,
-    setup_requires   = ['future'],
-    tests_require    = tests_require,
     install_requires = install_requires,
     extras_require   = {
         'tests': tests_require,
@@ -90,7 +85,6 @@
         'gpu': ['cupy', 'gputil', 'wurlitzer'],
         'optional': ['numexpr', 'mpldatacursor']},
     classifiers = [
-    'License :: OSI Approved :: BSD License',
     'Development Status :: 4 - Beta',
     'Intended Audience :: Education',
     'Intended Audience :: Science/Research',
